tests/system_test.py

```diff
@@ -488,6 +493,29 @@ def wait_ports(self, **retry_kwargs):
         wait_ports(self.ports_family, **retry_kwargs)
 
 
+class NginxServer(Process):
+    """
+    Sets up an ngix server
+    """
+    def __init__(self,
+                 config_path: str,  # Full path of templated config file (string) /blah-blah/nginx/nginx-template.conf
+                 env: dict[str, str],  # A dict of env variables which will be substituted in the config_path file.
```

Comment:
Actually, Python checker in CI is now a noop

```
No tests were found!!!
```

https://github.com/skupperproject/skupper-router/actions/runs/4009925749/jobs/6885845691#step:9:25

Reply:
> The type hint here is Python 3.9+, in prior versions you are supposed to do `Dict[str, str]`, https://docs.python.org/3.10/library/typing.html#typing.Dict
> 
> This does not matter for CI, apparently, because we don't run a Python typechecker, and if we wanted to run a type checker, we can always use Python 3.9+ for the type checking

Modified dict to typing.Dict

Reply:
> Actually, Python checker in CI is now a noop
> 
> ```
> No tests were found!!!
> ```
> 
> https://github.com/skupperproject/skupper-router/actions/runs/4009925749/jobs/6885845691#step:9:25

So the standalone Python checker job is not doing anything now ?

Reply:
> Modified dict to typing.Dict

I thought we are fine with Python 3.9+. And, I forgot, it might be actually problematic in lower python even if not typechecking because annotations are evaluated eagerly, There is `from __future__ import annotations` that is 3.7+  that optionally enables postponed evaluation of annotations https://peps.python.org/pep-0563/

```
$ python3.6

>>> dict[str,str]
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: 'type' object is not subscriptable
```

> So the standalone Python checker job is not doing anything now ?

Well, you saw it for yourself, no? `No tests were found!!!`